sys-fs/xwmfs/xwmfs-0.85.ebuild

```diff
@@ -0,0 +1,27 @@
+# Copyright 1999-2020 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+DESCRIPTION="FUSE-based X11 Window-Manager file system"
+HOMEPAGE="https://github.com/gerstner-hub/xwmfs"
+SRC_URI="https://github.com/gerstner-hub/${PN}/releases/download/v${PV}/${P}-dist.tar.gz -> ${P}.tar.gz"
+
+LICENSE="GPL-2"
+KEYWORDS="~amd64 ~arm ~x86"
+SLOT="0"
+
+RDEPEND="
+	sys-fs/fuse:0=
+	>=x11-libs/libX11-1.6.5"
+
+DEPEND="${RDEPEND}"
+BDEPEND="virtual/pkgconfig"
+
+src_test() {
+	# unset display to avoid testing errors, tests rely on X11 and a window
+	# manager environment
+	unset DISPLAY
```

Comment:
I wonder if using *virtualx.eclass* would be a better approach to this? Could you check it out sometime?